### PR TITLE
Fix / Flake8 fixes and cleanup (Goes out of date fast)

### PR DESCRIPTION
**Pull request by TheHolyRoger**
**Before submitting this PR, please make sure**:
- [X] Your code builds clean without any errors or warnings
- [X] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
A whole bunch of changes throughout most files to resolve flake8 errors. Finally get a full flake8 pass now.

Changes are almost all whitespace, this should be easy to review and merge before it goes out of date and rebase etc etc etc.!

`Flake8...................................................................Passed`

### CODE REVIEW

File: test/integration/test_eterbase_order_book_tracker.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from hummingbot.connector.exchange.eterbase.eterbase_order_book_tracker import EterbaseOrderBookTracker
 from hummingbot.connector.exchange.eterbase.eterbase_api_order_book_data_source import EterbaseAPIOrderBookDataSource
 from hummingbot.connector.exchange.eterbase.eterbase_order_book import EterbaseOrderBook
-.from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book import OrderBook
```
Comment (TheHolyRoger) [inline test/integration/test_eterbase_order_book_tracker.py:27]: Breaking typo here? Unless it's some cython magic I'm, unaware of :D

**Review by vic-en — APPROVED** [review]
LGTM

**Review by dennisocana — APPROVED** [review]
LGTM

**Comment by TheHolyRoger** [comment]
Thanks guys! And on latest development branch.....

`Flake8...................................................................Passed`

Woop :)

**Pull request merged**